source/template.yaml

```diff
@@ -0,0 +1,161 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Parameters:
+  gitsha:
+    Type: String
+    Description: Git sha 
+  environment:
+    Type: String
+    Description: Deployment environment
+    Default: "dev"
+
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: login-idp-functions
+    Description: login.gov idp functions
+    Author: logindotgov
+    SpdxLicenseId: CC0-1.0
+    HomePageUrl: https://github.com/18f/identity-idp-functions
+    SemanticVersion: 0.0.1
+    SourceCodeUrl: https://github.com/18f/identity-idp-functions
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      -
+        Label:
+          default: Application Settings
+        Parameters:
+          - gitsha
+          - environment
+
+Description: >
+  identity-idp-functions
+
+  IDP functions for login.gov
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+  Function:
+    Timeout: 60
+    Runtime: ruby2.7
```

Comment:
Ditto, can we make this 2.6?

Reply:
Or are we not able to? Looks like the docs only support 2.5 or 2.7? https://docs.aws.amazon.com/lambda/latest/dg/lambda-ruby.html